iredis/entry.py

```diff
@@ -55,11 +57,13 @@ def repl(client, session, start_time):
             _interval = None
 
         try:
+            get_rprompt = lambda: '<transaction>' if in_transaction else None
```

Comment:
Do we really need a lambda here?

Reply:
```suggestion
            get_rprompt = '<transaction>' if in_transaction else None
```

Reply:
I think this line should be out of try-catch scope. It can't generate any Exceptions.

Reply:
update